Upgrading to 6.2.1

biorus opened the pull request:
- Changed Liferay version
- Changed the category and the weight for control-panel-entry
- Added TLD dependencies in liferay-plugin-package.properties
- Removed the parameters yearRangeEnd and yearRangeStart from
  liferay-ui:input-date according to the new TLD
- Raised minute interval to 30

biorus committed:
Upgrading to 6.2.1

- Changed Liferay version
- Changed the category and the weight for control-panel-entry
- Added TLD dependencies in liferay-plugin-package.properties
- Removed the parameters yearRangeEnd and yearRangeStart from
liferay-ui:input-date according to the new TLD
- Raised minute interval to 30

akakunin commented on src/main/webapp/WEB-INF/liferay-plugin-package.properties:
```diff
@@ -8,7 +8,14 @@ page-url=http://emdev.ru
 portal-dependency-jars=\
     commons-lang.jar
-liferay-versions=6.1.1+,6.1.20+
+liferay-versions=6.2.1+,6.2.10+
```
should be 6.2.0+,6.2.10+

akakunin commented on pom.xml:
```diff
@@ -1,121 +1,121 @@
+	<version>6.2.1.1</version>
+
+	<properties>
+		<liferay.version>6.2.1</liferay.version>
```
I think better to use latest 6.2.3 here

akakunin commented:
Hi! Thank you for pull request. There are only two small comments to fix.

biorus commented:
Hi,

I'll change those but it'll take me a bit of time. I chose these version
because I'm mainly working with EE.

Regards,
Jose

2015-09-10 15:23 GMT+01:00 Alexey Kakunin notifications@github.com:

> Hi! Thank you for pull request. There are only two small comments to fix.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/emdev-limited/extented-auth-portlet/pull/2#issuecomment-139252526
> .

akakunin commented:
you do not need to switch to CE - just change numbers in code - it should be ok - I will test on CE from my side

biorus committed:
Updating of properties

- Changed the property liferay.version in liferay-plugin-package.properties
- Changed liferay version in pom.xml

Changes tested in 6.2 CE GA2, 6.2 CE GA4 and 6.2.10 EE GA1.
